src/db/models/projectVersionField.ts

```diff
@@ -0,0 +1,37 @@
+import * as t from 'io-ts';
+
+import { brandedType } from '../../util/io-ts';
+import { Brand } from '../../util/types';
+import { defineIDModel } from '../util/id-model';
+import { CONDITION_FIELD_ID } from './conditionField';
+import { PROJECT_VERSION_PLAN_ID } from './projectVersionPlan';
+
+export type ProjectVersionFieldId = Brand<
+  number,
+  { readonly s: unique symbol },
+  'projectVersionField.id'
+>;
+
+export const PROJECT_VERSION_FIELD_ID = brandedType<
+  number,
+  ProjectVersionFieldId
+>(t.number);
+
+export default defineIDModel({
+  tableName: 'projectVersionField',
+  fields: {
+    generated: {
+      id: { kind: 'branded-integer', brand: PROJECT_VERSION_FIELD_ID },
+    },
+    required: {
+      conditionFieldId: { kind: 'branded-integer', brand: CONDITION_FIELD_ID },
+      projectVersionPlanId: {
+        kind: 'branded-integer',
+        brand: PROJECT_VERSION_PLAN_ID,
+      },
+      value: { kind: 'checked', type: t.any },
```

Comment:
`value` is optional as it has no `NOT NULL` constraint and there are row which have `NULL` as value for this column (unlike `conditionFieldId` and `projectVersionPlanId`) above.

Also, please use `unknown` instead of `any` for json columns.
```suggestion
      value: { kind: 'checked', type: t.unknown },
```